src/scss/_custom.scss

```diff
@@ -1,16 +1,82 @@
 .leftcol {
-  background-color: blue;
+  // background-color: orange;
   float: left;
   clear: right;
   display: inline-block;
   width: 30%;
   margin: 10px;
+
+  img {
+    width: 230px;
+    height: 230px;
+    border-radius: 5px;
+  }
```

Comment:
Better to select the _specific_ element you're after here, rather than making _all_ `<img>` tags exactly 230x230px... Maybe?